### PR TITLE
GRW-842 / Move components from OfferNew to Offer

### PR DESCRIPTION
**Pull request by amandaflod**
## What?

Moving all components that were still used in Offer from OfferNew and delete OfferNew folder. Move utils and utils tests into corresponding files. 
## Why?

**Ticket(s): [GRW-842]**

[GRW-842]: https://hedvig.atlassian.net/browse/GRW-842?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/client/pages/Offer/Introduction/ExternalInsuranceProvider/index.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {
   useExternalInsuranceDataQuery,
   useExternalInsuranceDataStatusSubscription,
 } from 'data/graphql'
-import { OfferData } from 'pages/OfferNew/types'
+import { OfferData } from 'src/client/pages/Offer/types'
```
Comment (gustaveen) [inline src/client/pages/Offer/Introduction/ExternalInsuranceProvider/index.tsx:7]: I guess `src/client` should be removed here right? :) The auto import messes up the paths, a bit annoying :D 
```suggestion
import { OfferData } from 'pages/Offer/types'
```

**Review by gustaveen — COMMENTED** [review]
Awesome Amanda! It seems to be working but it's was a few places where the import path included `src/client` :)

File: src/client/utils/tracking/gtm/trackOfferGTM.ts [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import { datadogRum } from '@datadog/browser-rum'
-import { OfferData } from 'pages/OfferNew/types'
 import {
   isStudentOffer,
   hasHomeQuote,
   hasAccidentQuote,
   hasTravelQuote,
-} from 'pages/OfferNew/utils'
+} from 'pages/Offer/utils'
+import { OfferData } from 'src/client/pages/Offer/types'
```
Comment (gustaveen) [inline src/client/utils/tracking/gtm/trackOfferGTM.ts:8]: 👆

**Review by gustaveen — APPROVED** [review]
Found some more, sorry 😆 Not sure how picky we should be though 😄

**Comment by amandaflod** [comment]
Hahah oh no! Give me a minute and I'll get rid of them! Weird that it still works with them there? Usually everything fails 😅

**Pull request merged**